src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,22 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.3.0
+
+### New Feature
+
+- Git Catalog to provide predefined development environments to every developer
+- Add Git Catalog tab to Okteto UI deploy
+- Remove CLI tab from Okteto UI deploy
+
+### Bug Fix
+
+- Use case insensitive check for GitHub ID
+- GitHub redeploy modal was allowing `repo` field to be modified if opened fast enough
+- Support for "resources" for the okteto daemonset
+- Fix race condition that made namespaces in the Okteto UI empty  
+- Fix bug in the Okteto UI for the updated time of development environments
```

Comment:
What is the behavior being fixed here?

```suggestion
- Fix bug in the Okteto UI for the updated time of development environments
```

Reply:
@rberrelleza it was using the _creation_ date instead of the _last-updated_ date, so wrong value displayed to users

So how about:
```suggestion
- Fix wrong "Last updated" time for development environments in the Okteto UI
```

Reply:
ah I get it, looks good!